datadog_checks_dev/datadog_checks/dev/tooling/signing.py

```diff
@@ -117,7 +117,9 @@ def update_link_metadata(checks, core_workflow=True):
         key_id_prefix = signing_key['keyid'][:8].lower()
 
         tag_link = f'{STEP_NAME}.{key_id_prefix}.link'
-        options = {'signing_key': signing_key}
+        # Do not allow machine to normalize line endings, because it is not on Windows,
+        # for which git might rewrite line endings in files.
+        options = {'signing_key': signing_key, 'normalize_line_endings'=False,}
```

Comment:
CI is failing, I think this will fix

```suggestion
        options = {'signing_key': signing_key, 'normalize_line_endings'=False}
```